nessie/ops/.env

```diff
@@ -0,0 +1,3 @@
+TAG=latest
+LAKEFS_BLOCKSTORE_TYPE=local
+DOCKER_REG=977611293394.dkr.ecr.us-east-1.amazonaws.com
```

Comment:
Not part of this PR but I think we should look into having all this code and ops as code in a private repository as this code / test env is not public and managed on the public repository.

Reply:
Managing ops in a private repo has many disadvantages, let's discuss offline.

Reply:
Or (ab-)use "secret"s to store our specific information (like AWS account ID).